ui/ducks/send/send.js

```diff
@@ -1875,8 +1883,12 @@ export function updateRecipientUserInput(userInput) {
 
     const inputIsValidHexAddress = isValidHexAddress(userInput);
     let isProbablyAnAssetContract = false;
+    let smartContractAddress;
     if (inputIsValidHexAddress) {
-      const smartContractAddress = await isSmartContractAddress(userInput);
+      smartContractAddress = await isSmartContractAddress(userInput);
+      if (smartContractAddress) {
+        dispatch(actions.updateRecipientType(RECIPIENT_TYPES.SMART_CONTRACT));
+      }
       if (smartContractAddress) {
```

Comment:
is there a reason you made a new `if (smartContractAddress)` conditional? The same if statement is right below on line 1892?

Reply:
uh, no... I originally had this outside the if block two lines above, and then copy pasted it to here, will refactor

Reply:
changed in latest commit